fixes.inc

```diff
@@ -14281,6 +14411,242 @@ native BAD_DestroyActor(actorid) = DestroyActor;
 	#define DestroyActor__( FIXES_DestroyActor(
 #endif
 
+/**
+ * <fixes> Various bugs in GetVehicleComponentInSlot function.</fixes>
+ */
+
+#if _FIXES_SAMP && defined _ALS_GetVehicleComponentInSlot
+	#error _ALS_GetVehicleComponentInSlot defined
+#endif
+
+native BAD_GetVehicleComponentInSlot(vehicleid, slot) = GetVehicleComponentInSlot;
+
+#if FIX_GetVehicleComponentInSlot
+
```

Comment:
Since this is getting very close to acceptance (honestly, I'm willing to fix up the last bits since they're so minor if you want), I'm going to get picky.  There shouldn't be a blank line here, and all the `if`s should be followed by a space.